react-frontend/src/components/Home/cardList.js

```diff
@@ -100,6 +99,15 @@ const CardList = () => {
               routePath="/case-studies"
             />
           </Col>
+          <Col sm={12} md={8} lg={6} style={{ margin: 'auto' }}>
```

Comment:
I would stick keeping the columns styling the same as the others, that way it will match the figma designs. For a single card in row.
`<Col sm={12} md={6}>`
![image](https://user-images.githubusercontent.com/9705602/126202842-10f91518-c1c0-4acd-8f9b-f8343de63480.png)

Reply:
Thanks for keeping my scope creep in check. https://github.com/bcgov/digital.gov.bc.ca/pull/855/commits/d13e49c67974b66f85faa3eff16ef90969817e8b